configuration/structures.go

```diff
@@ -89,25 +89,23 @@ type BotWebhookAction struct {
 
 // PropertyConfig defines configuration of a property
 type PropertyConfig struct {
-	Type        string               `json:"type"`
-	Locations   map[string]*Location `json:"locations"`
-	Description string               `json:"description,omitempty"`
-	Domain      []interface{}        `json:"domain,omitempty"`
-	Range       *struct {
+	Name          string                     `json:"name"`
+	OwnerClientID string                     `json:"owner_client_id"`
+	Type          string                     `json:"type"`
+	Access        map[string]*PropertyAccess `json:"access"`
+	Description   string                     `json:"description,omitempty"`
+	Domain        []interface{}              `json:"domain,omitempty"`
+	Range         *struct {
 		From int `json:"from"`
 		To   int `json:"to"`
 	} `json:"range,omitempty"`
-}
-
-// Location represents property location
-type Location struct {
-	Access map[string]*PropertyAccess `json:"access"`
+	PublicAccess []string `json:"public_access,omitempty"`
```

Comment:
in JS sdk this is defined as `string`, not array - which one is correct? :D

Reply:
array